agent_deploy/kubernetes/sysdig-agent-daemonset-v2.yaml

```diff
@@ -65,13 +65,14 @@ spec:
         securityContext:
           privileged: true
         resources:
-          # Resources needed are subjective on the actual workload
-          # please refer to Sysdig Support for more info about it
+          # Resources needed are subjective on the actual workload,
+          # please contact Sysdig Support for tuning these numbers.
           requests:
-            cpu: 100m
+            cpu: 600m
             memory: 512Mi
           limits:
-            memory: 1024Mi
+            cpu: 2000m
+            memory: 1536Mi
```

Comment:
This is great but can you run this by Naveen? We talked about many different ways to do this, including having a small/medium/large example.

Reply:
Clearly, these larger numbers are better. But I'd recommend that we match the "small" config limits listed here: https://sysdig.atlassian.net/wiki/spaces/EN/pages/823984513/Agent+Resource+Allocation ?

We've started socializing the data in the above link with Support and having default numbers that don't match anything seems odd.